blocks/library/audio/index.js

```diff
@@ -98,7 +102,6 @@ registerBlockType( 'core/audio', {
 						<Button
 							className="components-icon-button components-toolbar__control"
 							aria-label={ __( 'Edit audio' ) }
-							type="audio"
```

Comment:
What was this accomplishing before?

Reply:
I think "nothing" as it's not a valid value for the "button" type attribute.